### PR TITLE
added iso27001 requirements to info-sec charter

### PR DESCRIPTION
**Pull request by andreas-kater**

### CODE REVIEW

File: handbook/information-security/README.md [hunk 1]
```diff
@@ -36,21 +39,179 @@ The framework includes programs covering:
...
+DVELP will take reasonable and proportionate actions to facilitate and assist
+law enforcement agencies (and other appropriately authorised investigators)
+pursuant of their duties and responsibilities, and in this respect shall make
+available in a timely manner all records, audit trails, system logs as may be
+requested by them by legal authority (i.e. subpoena, court order, search
+warrant, etc.).
+
+The legislation that DVELP has to comply with includes:
+*  General Data Protection Regulation (GDPR)
+*  Data Protection Act 1998
```
Comment (tom-mullen) [inline handbook/information-security/README.md:62]: Can we organise these alphabetically?

File: handbook/information-security/README.md [hunk 2]
```diff
@@ -36,21 +39,179 @@ The framework includes programs covering:
...
+## 4 Context of the organisation
+
+### Understanding the organization and its context
+
+DVELP is a software consultancy, building solutions for clients in a number of
+industries, including retail, health, and financial services. We are based from
+a headquarters in London, and maintain a remote team that operates
+internationally.
+
+### Understanding the needs and expectations of interested parties
```
Comment (tom-mullen) [inline handbook/information-security/README.md]: Can we make these titlecase?

File: handbook/information-security/README.md [hunk 3]
```diff
@@ -36,21 +39,179 @@ The framework includes programs covering:
...
+
+DVELP’s management team has identified the following stakeholders as being key
+to the design and maintenance of their information security management system:
+* Our customers (including end-users of the software solutions that are built by
+  DVELP)
+* Company Board
```
Comment (tom-mullen) [inline handbook/information-security/README.md:78]: Alpha

File: handbook/information-security/README.md [hunk 4]
```diff
@@ -36,21 +39,179 @@ The framework includes programs covering:
...
+* Staff
+* Contractors
```
Comment (tom-mullen) [inline handbook/information-security/README.md:79]: Does this need to include clients?

File: handbook/information-security/README.md [hunk 5]
```diff
@@ -36,21 +39,179 @@ The framework includes programs covering:
...
+<security@dvelp.co.uk> email address and evaluated within 24 hours.
+
+We assess risks in our business in line with our [Risk Management
+Policy](risk-management.md) and take action to improve our systems in accordance
+with our [Vulnerability Management Procedure](vulnerability-management.md).
+
+### Information security objectives and planning to achieve them
+
+The objectives of our Information Security Management System include:
+* continuously improving our processes by acting on reported vulnerabilities
```
Comment (tom-mullen) [inline handbook/information-security/README.md]: The other lists have the first letter capitalised.

File: handbook/information-security/README.md [hunk 6]
```diff
@@ -36,21 +39,179 @@ The framework includes programs covering:
...
-## Violations
+### Awareness
+
+We ensure our staff are aware of our policies by:
+* Ensuring new joiners read and understand our [Information Security
+  Policy](readme.md)
+* Regularly [auditing our compliance internally](internal-security-audit.md)
+* Regularly providing quizzes on our policies to all staff
+
+### Communication and Documented information
```
Comment (tom-mullen) [inline handbook/information-security/README.md]: Titlecase

File: handbook/information-security/risk-management.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+## Risk Management
+
+### Risk Register
+
+The company [Risk
+Register](https://docs.google.com/spreadsheets/d/1gyK0zWSVlX-ZnXsN_c4-my86-hHjHEDY6IEP5h9UuVQ/edit?usp=sharing)
+tracks all of the risks that members of the DVELP team have identified.
+
+Where the anticipated impact of an unmitigated risk exceeds a rating of 4, the
```
Comment (tom-mullen) [inline handbook/information-security/risk-management.md]: Can we detail the scale in this document?

**Review by tom-mullen — CHANGES_REQUESTED** [review]

File: handbook/information-security/README.md (hunk 4 above)
Comment (andreas-kater) [reply handbook/information-security/README.md:79]: It did under "Our customers". I've changed this to "Our Clients"

File: handbook/information-security/risk-management.md (hunk 7 above)
Comment (andreas-kater) [reply handbook/information-security/risk-management.md]: Now added

**Review by tom-mullen — APPROVED** [review]

**Pull request merged**